app/models/data_dot_gov_credentials.rb

```diff
@@ -0,0 +1,8 @@
+class DataDotGovCredentials
+  extend UserProvidedService
+
+  def self.api_key(force_vcap: false)
+    return ENV['DATA_DOT_GOV_API_KEY'] if use_env_var?(force_vcap)
```

Comment:
I personally find this format more readable for conditions:

```ruby
if use_env_var?(force_vcap)
  ENV['DATA_DOT_GOV_API_KEY']
else
  credentials('data-dot-gov')['api_key']
end
```

what do you think?